src/metax_api/tests/api/rest/base/views/datasets/write.py

```diff
@@ -77,32 +90,54 @@ def _get_new_test_data(self, cr_index=0, dc_index=0, c_index=0):
                     "@type": "Organization",
                     "name": {"fi": "Mysterious Organization"}
                 }
-            }],
-            "total_ida_byte_size": 1024,
-            "files": catalog_record_from_test_data['research_dataset']['files']
+            }]
         })
+
+        if dc['catalog_json']['research_dataset_schema'] == 'ida':
+            catalog_record_from_test_data['research_dataset'].update({
+                "total_ida_byte_size": 1024,
```

Comment:
sum(f['byte_size'] for f in research_dataset['files']) instead of hard coding 1024?